src/bp/core/repositories/telemetry_payload.ts

```diff
@@ -0,0 +1,125 @@
+import { ConfigProvider } from 'core/config/config-loader'
+import { inject, injectable } from 'inversify'
+import _ from 'lodash'
+import moment from 'moment'
+
+import Database from '../database'
+import { TYPES } from '../types'
+
+const DEFAULT_ENTRIES_LIMIT = 1000
+
+interface TelemetryEntries {
+  url: string
+  events: any[]
+}
+
+interface TelemetryEntry {
+  uuid: string
+  payload: any
+  available: boolean
+  lastChanged: Date
+  creationDate: Date
+}
+
+@injectable()
+export class TelemetryRepository {
+  private readonly tableName = 'telemetry'
+  private readonly telemetryServerUrl = process.TELEMETRY_URL
```

Comment:
this variable isn't necessary

Reply:
30d9f810ac3d343c5f27073b9cb2b27775b2c529